src/index.html

```diff
@@ -96,6 +96,7 @@ <h5>How you stack up: <span class="day-flights"></span>/<span class="users-avera
   <script type="text/javascript" src="../src/UserRepository.js"></script>
   <script type="text/javascript" src="../src/User.js"></script>
   <script type="text/javascript" src="../src/Hydration.js"></script>
+  <script type="text/javascript" src="../src/helper.js"></script>
   <script type="text/javascript" src="./scripts.js"></script>
```

Comment:
Since you're using the helper.js in the Hydration.js file, you'll probably want to import the `helper.js` before `Hydration.js`. This might be the reason you needed to check if the type of module was "undefined" in the `helper.js` file, because Hydration.js is trying to import it before it has been loaded.